src/app/zap-templates/zcl/data-model/chip/bridged-device-basic-information.xml

```diff
@@ -71,6 +71,12 @@ limitations under the License.
         <server tick="false" init="false">true</server>
         <globalAttribute side="either" code="0xFFFD" value="3"/>
 
+        <features>
+          <feature bit="20" code="BIS" name="BridgedICDSupport" summary="Support Bridged ICD Devices.">
```

Comment:
Typo on bit: it's 0, not 20.

Reply:
I was going to where the spec PR I am reference that I am working towards has specified. I am asking for clarification on the spec PR

https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9499/files#r1678479717